azurerm/internal/services/mssql/mssql_database_resource.go

```diff
@@ -558,6 +558,9 @@ func resourceMsSqlDatabaseRead(d *schema.ResourceData, meta interface{}) error {
 		d.Set("sku_name", props.CurrentServiceObjectiveName)
 		d.Set("storage_account_type", props.StorageAccountType)
 		d.Set("zone_redundant", props.ZoneRedundant)
+		if props.CreateMode != "" {
+			d.Set("create_mode", props.CreateMode)
+		}
```

Comment:
rather than doing this, can we instead put this in the Import function above, making that:

```go
		Importer: azSchema.ValidateResourceIDPriorToImportThen(func(id string) error {
			_, err := parse.DatabaseID(id)
			return err
		}, func(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
			d.Set("create_mode", "Default")
			if v, ok := d.GetOk("create_mode"); ok && v.(string) != "" {
				d.Set("create_mode", v)
			}
			
			return []*schema.ResourceData{d}, nil
		}),
```

which means this'll only be set at Import time

Reply:
Thanks, done! Normally I would also run the AccTests myself to validate, but I ran out of credits in my personal account, so that's up to you 😊